commit abc4a29156144641a285c0eec9b882c7e8f89530
Author: Yan, Zheng <zyan@redhat.com>
Date:   Thu Jan 19 11:21:29 2017 +0800

    ceph: update readpages osd request according to size of pages
    
    commit d641df819db8b80198fd85d9de91137e8a823b07 upstream.
    
    add_to_page_cache_lru() can fails, so the actual pages to read
    can be smaller than the initial size of osd request. We need to
    update osd request size in that case.
    
    Signed-off-by: Yan, Zheng <zyan@redhat.com>
    Reviewed-by: Jeff Layton <jlayton@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1359_abc4a29156144641a285c0eec9b882c7e8f89530.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1359_abc4a29156144641a285c0eec9b882c7e8f89530.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
